client/src/components/Header/components/exposed-nav.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { radioLocation } from '../../../../../config/env.json';
```

Comment:
sorry Link is deleted in my suggestion should have noticed it 😅 
```suggestion
import { radioLocation } from '../../../../../config/env.json';

import { Link } from '../../helpers';
```

Reply:
no worries.